.github/renovate.json

```diff
@@ -1,20 +1,12 @@
 {
   "$schema": "https://docs.renovatebot.com/renovate-schema.json",
   "extends": [
-    "config:best-practices",
-    ":enablePreCommit",
-    ":prConcurrentLimit20",
-    ":preserveSemverRanges",
-    ":semanticPrefixChore"
+    "github>scop/common"
   ],
   "ignorePresets": [
-    ":dependencyDashboard",
     ":ignoreModulesAndTests",
-    ":semanticPrefixFixDepsChoreOthers",
     "docker:pinDigests"
```

Comment:
Hm this did not work the way I thought it would, it appears `ignorePresets` was overridden rather than being merged, as the dep dashboard popped up again, https://github.com/scop/bash-completion/issues/589. Need to have a look.